### DISCUSSION TITLE
Bind own Window as ContextMenu

### DISCUSSION THREAD

**Discussion by steam3d** [HavenDV/H.NotifyIcon#79] [Q&A] [answered]
Can I bind my own window instead of `tb:TaskbarIcon.ContextFlyout` when I use `ContextMenuMode="SecondWindow"`?

Looks like no, but maybe I something miss

**Comment by HavenDV** [comment, marked as answer]
No, the code to create the second WinUI window is here: https://github.com/HavenDV/H.NotifyIcon/blob/ec46de75b99a375b190b662a21cb0b5cd3dc3da4/src/libs/H.NotifyIcon.Shared/TaskbarIcon.ContextMenu.cs#L241-L341
I can add such a possibility, but why is it?

**Reply by steam3d** [reply]
H.NotifyIcon already uses the second window. It means you could place any UI element inside and create a context menu like OneDrive has when you click on the tray icon. But it is limited programly, but it could give a better flexibility.

I am not forced you  to make it. Because I can create my own window and show it when the user clicks on the icon.

Anyway thanks for clarification.
